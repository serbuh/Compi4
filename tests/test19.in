// Nested while

int i = 8;
while(i > 0) {
  while(i > 0) {
    while(i > 0) {
      print i;
      i = i - 1;
    }
    print "End of inner-most while";
  }
  print "End of middle while";
}
print "End of outer-most while";
