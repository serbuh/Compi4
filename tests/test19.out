8
7
6
5
4
3
2
1
End of inner-most while
End of middle while
End of outer-most while
